olive/engine/packaging/packaging_generator.py

```diff
@@ -565,7 +572,42 @@ def _generate_onnx_mlflow_model(model_dir: Path, inference_config: Dict):
     return mlflow_model_path
 
 
-def _package_onnxruntime_packages(tempdir: Path, pf_footprint: "Footprint"):
+def _package_onnxruntime_genai_runtime_dependencies(tempdir: Path):
+    # pylint: disable=not-an-iterable
+    installed_packages = [
+        pkg
+        for pkg in pkg_resources.working_set
+        if pkg.key.startswith("onnxruntime-genai") or pkg.project_name.startswith("onnxruntime-genai")
```

Comment:
Any difference between key and project_name?

Reply:
I found some context here https://www.programcreek.com/python/?code=recipy%2Frecipy%2Frecipy-master%2Fintegration_test%2Fenvironment.py which gives the impression that they _could_ be different.

Reply:
Not a big deal for now, we could be more consistent in our code.